pkg/install/generate.go

```diff
@@ -140,24 +139,35 @@ func GenerateInstallerJob(
 		},
 	}
 
-	args := []string{"cluster"}
-
+	// This container just needs to copy the required install binaries to the shared emptyDir volume,
+	// where our container will run them. This is effectively downloading the all-in-one installer.
 	containers := []corev1.Container{
 		{
 			Name:            "installer",
 			Image:           installerImage,
 			ImagePullPolicy: installerImagePullPolicy,
 			Env:             env,
-			Args:            args,
+			Command:         []string{"/bin/sh", "-c"},
+			Args:            []string{"cp -v /bin/openshift-install /output && cp -v /bin/terraform /output && ls -la /output"},
+			VolumeMounts:    volumeMounts,
+		},
+		{
+			Name:            "hive-manager",
+			Image:           "hive-controller",
```

Comment:
We should allow you to configure this image (as well as the installer one)

Reply:
Do we think we'll just have one hive image to run, for multiple possible versions of the installer image?

If so then we could probably still get by with what we last talked about, just specifying the installer image to use directly on your cluster deployment, and presumably the hive image would be configured as part of the deployment template or something like that.

If not, then I think we might be talking ClusterVersion again.